byway/src/main/java/com/google/sps/servlets/DestinationsServlet.java

```diff
@@ -40,10 +39,10 @@ public void init() {
     datastore.put(userEntity);
     userKey= userEntity.getKey();
   }
-
+  
```

Comment:
Don't have a "return" for a void method.

Reply:
Zach told me to have an empty return statement so that I don't have to worry about entity being null because the entity will always have a value if it reaches the code after the catch

Reply:
Oh okay, I got a comment from Zach before telling me to delete the return so I thought it was the same for all cases.

Reply:
No empty return at the *end* of the method. If you want to return from a method early, like in an `if` condition, you can use an empty return in a void function, but there's no reason to put an empty return at the bottom.